sambaplayersdk/src/main/java/com/sambatech/player/cast/CastPlayer.java

```diff
@@ -159,8 +164,14 @@ else if (jsonObject.has("type")) {
           jsonObject = new JSONObject(message);
           String type = jsonObject.getString("type");
 
-          if (type.equalsIgnoreCase("finish"))
-            sambaCast.stopCasting();
+          if (type.equalsIgnoreCase("finish")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "finish" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.sambatech.com/coding_rules#rule_key=squid%3AS1132)